packages/common/src/utils/configSchema.ts

```diff
@@ -8,13 +8,15 @@ const contentSchema: SchemaOf<Content> = object({
   featured: boolean().notRequired(),
   backgroundColor: string().nullable().notRequired(),
   type: mixed().oneOf(['playlist', 'continue_watching', 'favorites', 'content_list']),
+  custom: object(),
```

Comment:
The validation doesn't match the typings, is `custom` always defined in the response?

If not:

```suggestion
  custom: object().optional(),
```

If it can also be null:

```suggestion
  custom: object().notRequired(), // same as `object().nullable().optional()`
```